beacon_node/store/src/config.rs

```diff
@@ -62,18 +62,19 @@ pub struct StoreConfig {
 
 /// Variant of `StoreConfig` that gets written to disk. Contains immutable configuration params.
 #[superstruct(
-    variants(V1, V22),
+    variants(V1, V22, V24),
     variant_attributes(derive(Debug, Clone, PartialEq, Eq, Encode, Decode))
 )]
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct OnDiskStoreConfig {
     #[superstruct(only(V1))]
     pub slots_per_restore_point: u64,
     /// Prefix byte to future-proof versions of the `OnDiskStoreConfig` post V1
-    #[superstruct(only(V22))]
+    #[superstruct(only(V22, V24))]
     version_byte: u8,
-    #[superstruct(only(V22))]
+    #[superstruct(only(V22, V24))]
     pub hierarchy_config: HierarchyConfig,
+    // TODO(hdiff): Should persist the hot hierarchy_config too?
```

Comment:
yes

Reply:
we decided to keep this the same for hot and cold 7d8343f9e6f2db61b2c4fbc625bf1d3dd8f8d91b